tests/integration/cqlengine/query/test_queryset.py

```diff
@@ -602,6 +602,8 @@ def test_distinct_with_explicit_count(self):
 @requires_collection_indexes
 class TestQuerySetOrdering(BaseQuerySetUsage):
     @execute_count(2)
+    @xfail_scylla(
+        reason="Scylla does not support ordering on non-primary key columns: https://github.com/scylladb/python-driver/issues/343")
     def test_order_by_success_case(self):
         q = TestModel.objects(test_id=0).order_by('attempt_id')
         expected_order = [0, 1, 2, 3]
```

Comment:
Do we have a core issue for this? If not, maybe we should open one?

Reply:
It turned out there was bug in `AbstractQuerySet.order_by`, that triggered scylla-specific behavior.
I have fixed it there and now test works just fine, please take a look